app/tests/test_models.py

```diff
@@ -916,7 +933,7 @@ def test_user_get_most_complete_profiles_works(self):
         db.session.commit()
         self.assertEquals(models.Email.query.count(), 4)
         self.assertEquals([(u.email, score) for u, score in models.User.get_most_connected_profiles()],
-                          [(self.sly_less.email, 3),
-                           (dubya.email, 2),
-                           (lennon.email, 2),
-                           (stone.email, 1)])
+                          [(self.sly_less.email, 3L),
+                           (lennon.email, 2L),
+                           (dubya.email, 2L),
```

Comment:
The middle two users were swapped, which makes me wonder if this method needs an additional ordering or something, so that the results here aren't subject to the backend we're using.